README.md

```diff
@@ -30,6 +30,10 @@ handling the ACL and more.
 
 
 ## Supported OSes
+
+Linux kernel v4.13.0 or later is required.
```

Comment:
Just realized that this sentence is too restrictive.
This is true for the upstream ('vanilla') kernel, but distros may backport the driver to previous kernel versions.
Maybe something like `The required Linux kernel driver was upstreamed in v4.13`? Makes sense? Too technical?

Reply:
Maybe it could say "Linux kernel v4.13.0 includes the necessary hardware driver.  If using an older kernel then the standalone Thunderbolt driver version X.Y.Z or later needs to be installed as a module."

That should probably include a link to the driver too.  Does it seem accurate?